class.jetpack-gutenberg.php

```diff
@@ -128,15 +129,15 @@ protected static function share_items( $a, $b ) {
 	/**
 	 * Register a block
 	 *
-	 * @deprecated 7.1.0 Use jetpack_register_block() instead
+	 * @deprecated 7.1.0 Use \Automattic\Jetpack\jetpack_register_block() instead
```

Comment:
I'd love to avoid leading backslashes in every instance where we refer to namespaces or namespaced classes/functions/constants/entities.